### ISSUE TITLE
Custom animation in page transitions possible?

### ISSUE THREAD

**Issue by TrueGeek** [rid00z/FreshMvvm#199]
When transitioning between pages is it possible to specify the animation used? It defaults to slide up but I'd like to use slide in from left in certain cases and pop in for others.

Thanks!

**Comment by XplittR** [comment]
Did you close this issue because you figured out how to specify animation, or because you did not require it any more?

**Comment by TrueGeek** [comment]
Sorry, I still require the animation but I pulled down the source code and I saw it's just not as easy as passing in a parameter. I found this article and example code which shows how to do it:

https://github.com/jsuarezruiz/xamarin-forms-page-transitions
https://geeks.ms/jsuarez/2017/10/09/xamarin-forms-transiciones-entre-paginas/
(Google Translate required but it's readable)

It requires custom page renderers in the platform projects. I'm going to give it a go and if it works out I'll submit a PR.
